cli/src/android/update.ts

```diff
@@ -92,18 +102,19 @@ if (hasProperty('postBuildExtras')) {
 }
 
 export async function handleCordovaPluginsGradle(config: Config,  cordovaPlugins: Plugin[]) {
-  const pluginsFolder = resolve(config.app.rootDir, 'node_modules', '@capacitor/cli', 'assets', 'capacitor-android-plugins');
-  const pluginsGradlePath = join(pluginsFolder, 'build.gradle');
+  const pluginsFolder = await resolveNode(config, '@capacitor/cli', 'assets', 'capacitor-android-plugins');
+  const pluginsGradlePath = join(pluginsFolder!, 'build.gradle');
   let frameworksArray: Array<any> = [];
   let prefsArray: Array<any> = [];
   let applyArray: Array<any> = [];
   cordovaPlugins.map( p => {
+    const pluginPath = relative(p.rootPath, pluginsFolder!);
     const frameworks = getPlatformElement(p, platform, 'framework');
     frameworks.map((framework: any) => {
       if (!framework.$.type && !framework.$.custom) {
         frameworksArray.push(framework.$.src);
       } else if (framework.$.custom && framework.$.custom === 'true' && framework.$.type && framework.$.type === 'gradleReference') {
-        applyArray.push(`apply from: "../../../../${p.id}/${framework.$.src}"`);
+        applyArray.push(`apply from: "${pluginPath}/${framework.$.src}"`);
```

Comment:
Here `../../../../` is not the plugins folder, but `node_modules`. Then `${p.id}` is the specific cordova plugin folder that have the gradle file. So it should be something like `${resolveNode(p.id)}`.

Reply:
Ah, thanks ! So I have to resolve it in my patch. Thanks for the info.